packages/Modal/default/src/ModalCore.tsx

```diff
@@ -13,7 +13,7 @@ ReactModal.setAppElement('body');
 
 interface ModalCoreComponentProps {
   isOpen: boolean;
-  onOutsideTap: (event: MouseEvent | KeyboardEvent) => void;
+  onOutsideTap: (event: React.MouseEvent | React.KeyboardEvent) => void;
```

Comment:
Merci Arnaud, ne pouvais-tu pas juste destructurer ?
import React, { KeyboardEvent, MouseEvent } from 'react';

Reply:
Non je ne peux pas car il y a un pb TS (surement de conf) qui empêche ça et oblige import * as React.
Je pense que c'est potentiellement à cause de la version de Typescript

Reply:
tu as essayé l'option : esmoduleinterop ?

dispo depuis la version 2.7 : https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-7.html#support-for-import-d-from-cjs-from-commonjs-modules-with---esmoduleinterop

Reply:
Si je fais ça, je dois donc :
- soit le mettre dans le tsconfig du package et donc fixer tout le package
- soit le mettre au tsconfig global et changer tous les packages utilisant ts

Reply:
Ca fera l'objet d'une autre PR alors

Reply:
oui mieux faire cela dans une PR dédiée